### DISCUSSION TITLE
Office 2010 Updates

### DISCUSSION THREAD

**Discussion by themacintoshnerd** [LegacyUpdate/LegacyUpdate#31] [Questions & Help]
I installed LegacyUpdate on my XP VM which was installed using a SP3 ISO then updated with Service Pack 4 (the unofficial one) then installed my programs and then installed legacy update. The services works great as expected. However I have had one main issue. One of my Programs I installed was Office 2003.  Now LU did pick it up as Office 2003 but for whatever reason it's also telling me to install security updates for my non existent Office 2010. I have no idea why it picked up a Office 2010 install considering it also picked up the Office 2003 install. 

Windows XP x86 SP3, Internet Explorer 8.0
Update Agent 7.6.7600.256 - Supported version
ActiveX control 1.4.0.0 - Supported version

**Comment by kirb** [comment]
You can have Office components installed without having the full suite installed. For instance Word Viewer, Help Viewer (the built in help feature shared by Office and Visual Studio), or Office SDKs (also installed by Visual Studio). Also check add/remove programs, something might show up there. The product detections are Microsoft’s, usually based on presence of certain registry keys, often those that correspond to the add/remove programs entry.
